### ISSUE TITLE
Question regarding the state of the project

### ISSUE THREAD

**Issue by query-wow** [luismfonseca/age-of-mythology-private-server#1]
Hi, i wanted to ask ow far along is this, and are we able to establish a connection with the game?

**Comment by luismfonseca** [comment]
Yes, I think you can login and view join rooms. I don't know if those certificates I've generated are still valid though.

Do you plan to host this? :)

**Comment by query-wow** [comment]
Yes, I found a bunch of people that like the game. I don't know much about
go, I was wondering if I could port this to c# using your code base

**Comment by Introsider** [comment]
Dumb Question: is it also possible to make a server for aoe3, because it uses very similar structure as XML by example.
